packages/trader/src/sass/app/_common/drawer/contract-drawer.scss

```diff
@@ -73,6 +73,51 @@ $header-height: 4em;
 
 /** @define .contract-card; weak */
 .contract-card {
+    &--market-closed {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        padding: 0;
+        background-image: var(--contract-gradient-danger);
+        z-index: 1000;
+
+        p {
+            @include typeface(--paragraph-center-normal-black);
+            font-size: 12px;
+            color: var(--text-prominent);
+            text-transform: none;
+
+            &.market-is-closed-overlay__timer {
+                font-weight: bold;
+                font-size: 1.4rem;
```

Comment:
Do we still need this? Since we're using `Text` component already

Reply:
No. Removed them :+1: